src/Controls/samples/Controls.Sample.UITests/MauiProgram.cs

```diff
@@ -47,6 +47,11 @@ protected override void OnAppLinkRequestReceived(Uri uri)
 		protected override Window CreateWindow(IActivationState activationState)
 		{
 			var window = base.CreateWindow(activationState);
+			
+			window.Created += (s, e) =>
+			{
+				Issues.Issue17865.Instance?.RevealLastItem();
```

Comment:
Do we need to add this here? can't it be on the Issue Page ?

Reply:
In this way we reproduce exactly the same conditions as in the issue example. An attempt is made to scroll right after creating the Window and `ListViewBase` is not yet loaded.

Reply:
I would like to share an observation that might be important:

I encountered an issue which I suspected was related to a CollectionView not being loaded. To address this, I hooked into the OnLoaded event for CollectionView. My initial attempt was to process any pending scroll actions from a queue, but this approach resulted in the same exception, even though the CollectionView was already loaded.

Upon further investigation, I discovered that while the CollectionView might be loaded, its items might not yet be rendered. To tackle this, I utilized the ChildAdded event. I checked whether the BindingContext data type of the added child matched the item source's data type. Following this, I began counting the items: if the number of items added equaled the total items, I would then apply any pending scroll actions from the queue. This solution worked effectively, but it introduced a delay since the ChildAdded event takes time to add all child items.

I acknowledge that this workaround may not be ideal for large collections, but it has been effective in my case.